nebullvm/operations/inference_learners/builders.py

```diff
@@ -146,6 +146,25 @@ def execute(
         )
 
 
+class TensorFlowOpenVINOBuildInferenceLearner(BuildInferenceLearner):
```

Comment:
this class is actually useless. You can simply reuse the `OpenVINOBuildInferenceLearner` passing TensorFlow as `source_dl_framework`.